### ISSUE TITLE
Reading device info

### ISSUE THREAD

**Issue by miloush** [fadilfadz01/Universal_Updater#9]
Device running 10.0.10586.494, I get the updater stuck at

![image](https://github.com/fadilfadz01/Universal_Updater/assets/10546952/bc94713f-019e-4e22-94b9-b800e0647686)

any tips what to try?

**Comment by win98se** [comment]
I see “**Select** Universal Updater.exe” in the title bar. That means you're attempting to extract text from the black window.

In such interfaces, you're not supposed to click within the black window using a mouse; It'll suspend currently running operations.

You should use only a keyboard with this program.

**Comment by miloush** [comment]
Good catch but that is not the issue, the updater gets stuck and has to be killed. The new version seems to run into the same problem, although at least it times out properly:

![image](https://github.com/user-attachments/assets/45ee77b9-2344-42f8-a42d-85d0004cb1e9)

The device seems to be installed fine:
![image](https://github.com/user-attachments/assets/043b6867-e55b-4b73-b8c8-24fa6066cca2)

And I can browse the phone files in File Explorer.

**Comment by Cajnik59** [comment]
Hi guys, I have just the same issue but with Nokia Lumia 930. I am trying to update it to W10m and reading device info.... error 0x80070490. I have updated drivers with no any positive result. I can see and browse phone in file explorer.
Please help how to fix issue.

**Comment by miloush** [comment]
I successfully updated another phone (Lumia 735) from 10.0.14393.2551 to 10.0.15254.603 and now I it has the same issue, "Reading device info" no longer works.
